README.rst

```diff
@@ -233,7 +237,6 @@ After that, you can execute this folloing commands:
 
 ::
 
-    $ python server.py &
     $ export PYTHONPATH=$PWD
     $ nosetests -s -v --cover-package=facts --with-cover
```

Comment:
At this point (how to run unit tests), is it necessary show how to generate the coverage?